diff --git a/node_modules/@pmmmwh/react-refresh-webpack-plugin/client/ReactRefreshEntry.js b/node_modules/@pmmmwh/react-refresh-webpack-plugin/client/ReactRefreshEntry.js
index 980073b..ef82c65 100644
--- a/node_modules/@pmmmwh/react-refresh-webpack-plugin/client/ReactRefreshEntry.js
+++ b/node_modules/@pmmmwh/react-refresh-webpack-plugin/client/ReactRefreshEntry.js
@@ -1,6 +1,6 @@
 /* global __react_refresh_library__ */
 
-const safeThis = require('core-js-pure/features/global-this.js');
+var safeThis = require('core-js-pure/features/global-this.js');
 const RefreshRuntime = require('react-refresh/runtime.js');
 
 if (process.env.NODE_ENV !== 'production') {
@@ -10,6 +10,9 @@ if (process.env.NODE_ENV !== 'production') {
     if (typeof __react_refresh_library__ !== 'undefined' && __react_refresh_library__) {
       $RefreshInjected$ += '_' + __react_refresh_library__;
     }
+    if (safeThis.parent && safeThis.parent !== safeThis) {
+      safeThis = safeThis.parent
+    }
 
     // Only inject the runtime if it hasn't been injected
     if (!safeThis[$RefreshInjected$]) {
